resources/views/membership/showDetails.blade.php

```diff
@@ -0,0 +1,115 @@
+@extends('layouts.app')
+
+@section('content')
+<h1>New member detials review</h1>
+<p>Please review the details below and check they are all sane.</p>
```

Comment:
Maybe 'sane' isn't the most PC term.

"Please review the details below to check they are all correct"?